gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImplCreateTest.java

```diff
@@ -51,18 +63,32 @@
 @RunWith(Parameterized.class)
 public class GoogleCloudStorageImplCreateTest {
 
-  private static final String BUCKET_NAME = "foo-bucket";
```

Comment:
Is this used in integration test that we need to change bucket name?

Reply:
yeah, it's not required for unit tests.